adminservice/src/intTest/groovy/nu/yona/server/ActivityCategoriesTest.groovy

```diff
@@ -16,16 +16,65 @@ import spock.lang.Specification
 
 class ActivityCategoriesTest extends Specification
 {
+
 	// See https://docs.hazelcast.org/docs/latest/manual/html-single/index.html#near-cache-invalidation
 	// Default batch invalidation frequency is 10 seconds
-	static final def cachePropagationTimeoutSeconds = 240
+	static final def cachePropagationTimeoutSeconds = 20
 
 	@Shared
 	def AdminService adminService = new AdminService()
 
 	@Shared
 	def AppService appService = new AppService()
 
+	// NOTE: This test assumes that en-US is the client language
```

Comment:
Can't you [set the default locale](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Locale.html#setDefault(java.util.Locale)) under ``given:`` and then restore [the original](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Locale.html#getDefault()) under ``cleanup:``?

Reply:
Wouldn't it be better to pass an `Accept-Language` header with all requests in our acceptance test by default?

Reply:
That works for most of the tests, but some tests (``LocalizationTest`` and ``PinResetRequestTest``) explicitly set that header, so I'd rather only change those tests that care.

Reply:
I fixed a couple of tests that failed because of the different client locale, but there are still 54 other tests failing on my Dutch system. For example, some are related to the fact that the sender of a message is expected to be "RQ (me)" but is "RQ (ik)" (`ActivityCommentTest`, `AppActivityTest`, `BasicBuddyTest`, `RemoveUserTest`). Others are related to the fact that a different invite email is sent (`CreateUserOnBuddyRequestTest`). Yet others are related to different messages (`DeviceManagementTest`, `RemoveUserTest`, `UpdateBuddyUserInfoTest`, `UserPhotoTest`, `VpnConnectionStatusEventTest`). Yet others are related to the name of the default device (`OverwriteUserTest`).

So I still think we should make it default in the tests that we use `Accept-Language: en-US` and override that default by specific tests. But preferably in another PR.

Reply:
I didn't know the impact is that large. That should indeed go in a different PR. Setting ``en-US`` as default for the tests seems the right approach indeed.

The comment line in the code that this GitHub comment is attached to can be removed now, right?

Reply:
Removed the comment